### DISCUSSION TITLE
Logs Explorer error

### DISCUSSION THREAD

**Discussion by richleach** [supabase/supabase#11175] [Questions] [answered]
I'm trying to troubleshoot a query in my Next.js app and want to see what Supabase is receiving from my app. I'm not getting any data, nor any errors, so in the Logs Explorer I'm trying to run the canned query:
select
  cast(t.timestamp as datetime) as timestamp,
  p.error_severity,
  event_message
from postgres_logs as t
  cross join unnest(metadata) as m
  cross join unnest(m.parsed) as p
where
  p.error_severity in ('ERROR', 'FATAL', 'PANIC')
order by
  timestamp desc
limit 100
postgres_logs

But I get the error:
syntax error at line 13 column 1 near "postgres_logs" in "[postgres_lo]"

Has anyone seen this error before? Anyone know how to get around it? 

Thanks in advance, 
Rich

**Comment by Ziinc** [comment, marked as answer]
Hi try to run the following:
```sql
select
cast(t.timestamp as datetime) as timestamp,
p.error_severity,
event_message
from postgres_logs as t
cross join unnest(metadata) as m
cross join unnest(m.parsed) as p
where
p.error_severity in ('ERROR', 'FATAL', 'PANIC')
order by
timestamp desc
limit 100
```
The syntax error looks like you had inserted the  `postgres_logs` source at the bottom.
